format/spec.md

```diff
@@ -951,24 +958,45 @@ Types are serialized according to this table:
+| **`map`**                    | `JSON object: {`<br />&nbsp;&nbsp;`"type": "map",`<br />&nbsp;&nbsp;`"key-id": <key id int>,`<br />&nbsp;&nbsp;`"key": <type JSON>,`<br />&nbsp;&nbsp;`"value-id": <val id int>,`<br />&nbsp;&nbsp;`"value-required": <bool>`<br />&nbsp;&nbsp;`"value": <type JSON>`<br />`}`                                                                                                                                                                                                                              | `{`<br />&nbsp;&nbsp;`"type": "map",`<br />&nbsp;&nbsp;`"key-id": 4,`<br />&nbsp;&nbsp;`"key": "string",`<br />&nbsp;&nbsp;`"value-id": 5,`<br />&nbsp;&nbsp;`"value-required": false,`<br />&nbsp;&nbsp;`"value": "double"`<br />`}`                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
+
+For default values, the serialization depends on the type of the corresponding column or nested field. The mapping of types and their corresponding default value JSON serialization is described in the following table:
+
+| Type               | Json type         | Example                          | Note                                                                                                                                                                                                                                                                                                                      |
```

Comment:
I think it would be better to match the conventions used by the table above, including the column headings and type references, like `JSON int`.